pkg/drivers/kic/oci/oci.go

```diff
@@ -576,3 +579,19 @@ func ShutDown(ociBin string, name string) error {
 	glog.Infof("Successfully shutdown container %s", name)
 	return nil
 }
+
+// iptablesFileExists checks if /var/lib/dpkg/alternatives/iptables exists in minikube
+// this file is necessary for the entrypoint script to pass
+// see: https://github.com/kubernetes/minikube/issues/8179
+func iptablesFileExists(ociBin string, nameOrID string) bool {
+	if ociBin != Docker {
```

Comment:
This would be same for both Docker and Podman

Reply:
cool, fixed